src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -292,7 +293,7 @@ def default_api_version(self):
         ResourceType.MGMT_SERVICEBUS: '2021-06-01-preview',
         ResourceType.MGMT_IOTHUB: '2019-07-01-preview',
         ResourceType.MGMT_DATABOXEDGE: '2019-08-01',
-        ResourceType.MGMT_CONTAINERREGISTRY: '2019-05-01',
```

Comment:
are we sure we wanted to bump this one?

Reply:
spoke offline, it is better if we keep this azurestack hybrid one the current GA version